packages/wordcount/src/defaultSettings.js

```diff
@@ -1,3 +1,32 @@
+/**
+ * @typedef {import("./index").WordCountStrategy} WordCountStrategy
+ */
+
+/**
+ * @typedef {Object} WordCountSettingsFields
```

Comment:
It's currently promoted that our custom types be prefixed with `WP`:

```suggestion
 * @typedef {Object} WPWordCountSettingsFields
```

>A `WP` prefix should be applied to all custom types.

https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#custom-types

Personally, I've been waffling on this a bit lately. The general thinking was to avoid conflicts or possible ambiguity with types inherited from "global" namespace, where here without a prefix, there's a possibility it leaves room for misinterpreting this as some default type, vs. custom to our packages.

Reply:
Oh, thanks, missing the _first_ section of the guidelines is almost embarrassing :) Fixed in d098351. 

P.S. VSCode's rename (F2 for me) worked great. When importing had to rename both types, though. I wish that `/** @typedef {import('./lib').Type} */` worked and implied the type name without having to repeat it.